### PR TITLE
Dialogflow1

### PR DESCRIPTION
**Pull request by Supermn54**
image

Working Chat Bot (aka videBot) - Integrated Dialotflow into Chat App.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -219,6 +220,53 @@ app.get("/auth/google/callback",
     }
 );
 
+//sends text to dialogflow
+app.post("/sendToDialogflow", function(req, res){
+	//create variables
+	const projectId = 'helloworld-b026e';
+	const sessionId = '123';
+	const query = req.body["content"];
+	//const query = req.body["content"];
```
Comment (bonhamjp) [inline main.js]: I think you can get rid of this commented out code, since it is un-commented above

**Review by bonhamjp — COMMENTED** [review]
It looks like you included a .swp file, which I think is like a vim history file?

**Review by wollmank — APPROVED** [review]

File: main.js (hunk 1 above)
Comment (Supermn54) [reply main.js]: good catch....yeah it was there from when i was testing different values for the query variable. i'll update that and push it to this PR.

**Comment by Supermn54** [comment]
@bonhamjp yeah that swp file was the result of me having open files via vim on accident. I don't see it any more. Did you delete it?

**Review by bonhamjp — APPROVED** [review]
Awesome thanks!

**Review by wollmank — APPROVED** [review]

**Pull request merged**